client/components/command-pallette/use-command-pallette.tsx

```diff
@@ -1,4 +1,5 @@
 import styled from '@emotion/styled';
+import React from 'react';
```

Comment:
@katinthehatsite , do we need to import react?

Reply:
Done!